diff --git a/gradience/backend/theming/shell.py b/gradience/backend/theming/shell.py
index 18e7890f..fc83eb3f 100644
--- a/gradience/backend/theming/shell.py
+++ b/gradience/backend/theming/shell.py
@@ -21,6 +21,7 @@ import re
 import shutil
 import os.path
 import sass
+import stat
 
 from gi.repository import GObject, Gio, GLib
 
@@ -83,13 +84,21 @@ class ShellTheme:
         self.templates_dir = os.path.join(datadir, "gradience", "shell", "templates", version_target_str)
         self.source_dir = os.path.join(GLib.get_home_dir(), ".cache", "gradience", "gradience-shell", version_target_str)
 
-        if os.path.exists(self.source_dir):
-            shutil.rmtree(self.source_dir)
+        def copy_function(src, dst):
+            dst_dir = os.path.dirname(dst)
+            os.chmod(dst_dir, os.stat(dst_dir).st_mode | stat.S_IWRITE)
+
+            if (os.path.exists(dst)):
+                os.remove(dst)
+
+            shutil.copy2(src, dst)
+            os.chmod(dst, os.stat(dst).st_mode | stat.S_IWRITE)
 
-        # Copy shell theme source directories to ~/.cache/gradience/gradience-shell
         shutil.copytree(
-            os.path.join(datadir, "gradience", "shell", version_target_str),
-            self.source_dir, dirs_exist_ok=True
+            os.path.join(datadir, "gradience", "shell", str(self.version_target)),
+            self.source_dir,
+            dirs_exist_ok=True,
+            copy_function=copy_function
         )
 
         # TODO: Allow user to use different name than "gradience-shell" (also, with default name, we should append "-light" suffix when generated from light preset)
