diff --git a/src/main.rs b/src/main.rs
index 6b1a044..a9474ae 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -33,18 +33,37 @@ const LEGACY_USER_CONFIG: &str = ".safe-rm";
 const REAL_RM: &str = "/bin/rm";
 
 const DEFAULT_PATHS: &[&str] = &[
+    "/",
     "/bin",
     "/boot",
     "/dev",
     "/etc",
+    "/etc/ssh",
+    "/etc/ssh/ssh_host_ed25519_key",
+    "/etc/ssh/ssh_host_rsa_key",
     "/home",
+    "/home/nat",
+    "/home/nat/.ssh",
+    "/home/nat/.ssh/id_ed25519",
+    "/home/natwork",
+    "/home/natwork/.ssh",
+    "/home/natwork/.ssh/id_ed25519",
     "/initrd",
     "/lib",
     "/lib32",
     "/lib64",
+    "/lost+found",
+    "/mnt",
+    "/nix",
+    "/nix/store",
+    "/persist",
+    "/persist",
+    "/persistent",
     "/proc",
     "/root",
+    "/run",
     "/sbin",
+    "/srv",
     "/sys",
     "/usr",
     "/usr/bin",
@@ -59,6 +78,12 @@ const DEFAULT_PATHS: &[&str] = &[
     "/usr/share",
     "/usr/src",
     "/var",
+    "/var/lib",
+    "/vol",
+    "/vol/blahaj",
+    "/vol/games",
+    "/vol/Games",
+    "/vol/ikea",
 ];
 
 const MAX_GLOB_EXPANSION: usize = 256;
